### PR TITLE
feature airgap for mgmt-cluster

### PR DESCRIPTION
**Pull request by alknopfler**
- Add airgap scenario for mgmt-cluster

### CODE REVIEW

**Comment by alknopfler** [comment]
Tested using a libvirt network without connectivity:
```
+---------------+--------+------------------+------+---------------+----------+
| Network       |  Type  |       Cidr       | Dhcp |     Domain    |   Mode   |
+---------------+--------+------------------+------+---------------+----------+
| airgapnetwork | routed | 192.168.100.0/24 | True | airgapnetwork | isolated |
```
and after provisioning all pods running, so images has been deployed from the airgap solution without any external connection: 
```
mgmt-cluster-network:/etc/systemd/system # kubectl get pods -A|wc -l
68
mgmt-cluster-network:/etc/systemd/system # kubectl get pods -A -ojson | jq '.items | map(select(.status.phase = "Running")) | length '
68
```

**Comment by alknopfler** [comment]
Also it has been tested on a multi-node environment with metallb working fine

File: telco-examples/mgmt-cluster/airgap/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+```
+
+> **_IMPORTANT:_**  
+> if you want to use a different version of the `cluster-api`, `cluster-api-provider-rke2` and `cluster-api-provider-metal3` repositories, you need to modify the curl commands to download the necessary files.
+
+A list of curl commands to download the necessary files to be used in the air-gap scenario is provided:
+
+```bash
+# clusterctl binary
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/1.6.2/clusterctl-linux-${GOARCH} -o /usr/local/bin/clusterct
```
Comment (hardys) [inline telco-examples/mgmt-cluster/airgap/README.md:105]: Users won't necessarily have GOARCH defined in their environment and we currently only support the EIB step on x86 so perhaps replace `${GOARCH}` with [amd64](https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.2/clusterctl-darwin-amd64) ?

Also the -o name is truncated to `clusterct`

**Comment by hardys** [comment]
lgtm but small comment added - also if you address https://github.com/suse-edge/suse-edge.github.io/pull/307#discussion_r1660946392 we'll need to align the logic here.

I also wonder if we can avoid adding [airgap-resources.tar.gz](https://github.com/suse-edge/atip/pull/1/files#diff-2090903b669f7caaba6c42eb3c1e680a4427692520d7c0ed7f0a14d3bf38b009) to the git repo, as once we add it, it will be in the git history forever and it's a large file - could we perhaps add a script to create it instead?

File: telco-examples/mgmt-cluster/airgap/README.md (hunk 1 above)
Comment (alknopfler) [reply telco-examples/mgmt-cluster/airgap/README.md:105]: good catch! I will change it. Thanks

**Comment by alknopfler** [comment]
@hardys you're right! I've removed the tarball from the history, and I've explained in the README with the curls and the providers content to generate the tarball.
Thanks for your review

**Review by hardys — APPROVED** [review]

**Pull request merged**